AcademyResidentInformationApi.Tests/AcademyResidentInformationApi.Tests.csproj

```diff
@@ -1,26 +1,25 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
-    <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
```

Comment:
Does this still need to be added back in?

Reply:
Thanks for spotting this. I missed this while I was rebasing. 
Will add this back in.

Reply:
Have added this in 86dd418